docs/generate.rst

```diff
@@ -0,0 +1,126 @@
+Generation of benchmarks
```

Comment:
be careful documenting options like this. It makes it very easy to have outdated documentation with changes to the CLI interface